unlock-app/src/components/interface/checkout/main/Confirm.tsx

```diff
@@ -158,7 +158,15 @@ export function Confirm({
 
   const { isInitialLoading: isInitialDataLoading, data: purchaseData } =
     useQuery(
-      ['purchaseData', lockAddress, lockNetwork, JSON.stringify(recipients)],
+      [
+        'purchaseData',
+        lockAddress,
+        lockNetwork,
+        JSON.stringify(recipients),
```

Comment:
Remove the JSON.stringify actually. React query handle arrays on its own so  don't have to do this unlike react hooks.